src/daft-sql/src/planner.rs

```diff
@@ -246,6 +246,17 @@ impl SQLPlanner {
                     .collect::<SQLPlannerResult<Vec<_>>>()?;
             }
         }
+        let sort_exprs = query
+            .order_by
+            .clone()
+            .map(|order_by| {
+                if order_by.interpolate.is_some() {
+                    unsupported_sql_err!("ORDER BY [query] [INTERPOLATE]");
+                }
+                // TODO: if ordering by a column not in the projection, this will fail.
```

Comment:
better todo, make issue if you haven't already and include issue number in comment I'm thinking?

also kinda confused are you saying 
- it fails now and you don't want it to
- it doesn't fail and you want it to